content/ecosystem/_index.html

```diff
@@ -0,0 +1,461 @@
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> MACHINE LEARNING</span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white"> cuML is a library for executing machine learning algorithms on GPUs with an API
+            that closely follows the Scikit-Learn API. Reduce the time to fit your model from hours or minutes to seconds.</p>
+          <a href="https://github.com/rapidsai/cuml">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cuml/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+          <br><br>
+
+          <h3 class="has-text-white">libcuml</h3>
+          <p class="has-text-white">libcuml provides the CUDA C++ building blocks powering cuML.</p>
+          <a href="https://github.com/rapidsai/cuml">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/libcudf/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i>
+          </a>
+        </div>
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-terminal"></i> cuGraph</h2>
+          <div class="tags-container border-top has-border-white">
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> NETWORKX </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> GRAPH </span>
+            <span class="tags-item has-text-white has-border-white"><i class="fa-solid fa-tag"></i> PYTHON / C++ </span>
+          </div>
+          <p class="has-text-white"> cuGraph is a library to make graph analytics available to everyone. With a Python
+            API that follows NetworkX, you can analyze networks of millions of nodes without specialized software faster
+            than ever.</p>
+          <a href="https://github.com/rapidsai/cugraph">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/cugraph/stable/">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+
+          <br><br>
+
+          <h3 class="has-text-white">libcugraph</h3>
+          <p class="has-text-white">libcugraph provides the CUDA C++ building blocks powering cuGraph.</p>
+          <a href="https://github.com/rapidsai/cugraph">GitHub <i class="fa-solid fa-arrow-up-right"></i> </a> <br>
+          <a href="https://docs.rapids.ai/api/libcugraph/stable">Documentation <i class="fa-solid fa-arrow-up-right"></i> </a>
+        </div>
+      </div>
+      <div class="columns">
+        <div class="column is-one-third">
+          <h2 class="has-text-white border-bottom-white"><i class="fa-regular fa-terminal"></i> cuSpatial</h2>
```

Comment:
cuSpatial is in pretty heavy development and is EA right now--should we swap its order with RAFT?

Reply:
Not really in any particular order - Its cuspatial has the lib section so I kept it left aligned.